src/components/Summary/SummaryDefineCompany.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+    <v-card flat>
+      <div v-if="!defineCompanyStepValid" class="defineCompanyStepErrorMessage">
+         <span>
+          <v-icon color="#1976d2">mdi-information-outline</v-icon>
+          This step is not complete.
+          <router-link :to="{path:'/define-company', query: {showErrors: true}}">
+             Return to this step to complete it.
+          </router-link>
+         </span>
+      </div>
+      <div class="business-contact-container">
+        <BusinessContactInfo
+        :initialValue = businessContact
+        :isEditing = false />
+      </div>
+    </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Vue } from 'vue-property-decorator'
+import { Getter, Action, State } from 'vuex-class'
+
+// Interfaces
+import { BusinessContactIF } from '@/interfaces'
+
+// Components
+import { BusinessContactInfo } from '@/components/DefineCompany'
+
+@Component({
+  components: {
+    BusinessContactInfo
+  }
+})
+export default class SummaryDefineCompany extends Vue {
+  // State
+  @State(state => state.stateModel.defineCompanyStep.defineCompanyStepValid)
+  readonly defineCompanyStepValid!: boolean
```

Comment:
This is pretty neat. Is this a way of deconstructing the model at the same time as binding it?

Reply:
Yes :)